tests/views/test_survey_response_view.py

```diff
@@ -46,14 +47,17 @@ def test_add_survey_response_success(self):
             # When that file is post to the survey response end point
             response = self.client.post(
                 '/survey_response-api/v1/survey_responses/{case_id}'.
```

Comment:
it's minor but rewrite this to `f'/survey_response-api/v1/survey_responses/{case_id}'` so we can remove the .format